reference/auth.v1.yaml

```diff
@@ -680,6 +680,159 @@ paths:
         - sessionToken: []
       tags:
         - Internal
+  /v1/consents:
+    get:
```

Comment:
Do we need to be able to restrict the list of consent types that _could_ be accepted by a user? Or can everyone consent to everything? Put another way: does this API require a valid access token?

I could see a _potential_ use-case for restricting this due to kind of an invitation-only or restricted clinical trial. However, this creates a problem with defining who can see which consent types, and how does that work with people who do not yet have a user account. Similar to invitation workflows.

Reply:
The list endpoint is for internal/admin use only. We could make this endpoint public and add restrictions in the future. The frontend should be responsible for fetching a particular consent by type so it can be rendered.